File: jamcatch/jamcatch-stage/src/testEndToEnd/java/org/example/jamcatch/stage/end2end/JamCatchStageEnd2EndTest.java
package org.example.jamcatch.stage.end2end;

import static org.junit.jupiter.api.Assertions.assertTrue;

import org.example.javarca.engine.test.fixtures.RendererFixture;
import org.example.javarca.engine.test.fixtures.Screenshot;
import org.junit.jupiter.api.Tag;
import org.junit.jupiter.api.Test;

@Tag("slow")
public class JamCatchStageEnd2EndTest {

    @Test
    public void test() {
        Screenshot result = RendererFixture.launchAndWaitForFinish();

        assertTrue(result.file().exists(), "Screenshot should exist");
    }
}

File: jamcatch/jamcatch-stage/src/testEndToEnd/kotlin/org/example/jamcatch/stage/end2end/JamCatchStageEnd2EndTest.kt
package org.example.jamcatch.stage.end2end

import org.example.javarca.engine.test.fixtures.RendererFixture
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Tag
import org.junit.jupiter.api.Test

@Tag("slow")
class JamCatchStageEnd2EndTest {
    @Test
    fun test() {
        val result = RendererFixture.launchAndWaitForFinish()

        Assertions.assertTrue(result.file.exists(), "Screenshot should exist")
    }
}
